l10n_it_account_stamp/i18n/it.po

```diff
@@ -0,0 +1,109 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
+# 	* l10n_it_account_stamp
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenERP Server 7.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-12-05 15:25+0000\n"
+"PO-Revision-Date: 2018-12-05 16:31+0100\n"
+"Last-Translator: <>\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"Language: it\n"
+"X-Generator: Poedit 1.8.12\n"
+
+#. module: l10n_it_account_stamp
+#: code:addons/l10n_it_account_stamp/invoice.py:73
+#, python-format
+msgid "Missing account configuration for %s"
+msgstr "Configurazione mancante sui conti di ricavo/costo per il prodotto %s"
+
+#. module: l10n_it_account_stamp
+#: field:product.template,stamp_apply_min_total_base:0
+msgid "Stamp apply min total base"
+msgstr "Base imponibile per l'applicazione"
+
+#. module: l10n_it_account_stamp
+#: view:product.product:0
+msgid "{'readonly': [('is_stamp', '=', True)]}"
+msgstr ""
+
+#. module: l10n_it_account_stamp
+#: constraint:product.template:0
+msgid "The product must be a stamp to set apply taxes!"
+msgstr "Il prodotto deve essere una marca da bollo per poter impostare i conti imposta!"
+
+#. module: l10n_it_account_stamp
+#: view:product.product:0
+msgid "For each invoice, the base amount for each selected tax code account will be added up and used to determine the application of the account stamp."
+msgstr "La base imponibile di ogni conto imposta individuato fra quelli sopra elencati verrà sommata e utilizzata, all'interno di ciascuna fattura, per determinare l'applicabilità della marca da bollo."
+
+#. module: l10n_it_account_stamp
+#: model:ir.model,name:l10n_it_account_stamp.model_product_template
+msgid "Product Template"
+msgstr "Template di prodotto"
+
+#. module: l10n_it_account_stamp
+#: view:product.product:0
+msgid "{'required': [('is_stamp', '=', True)]}"
+msgstr ""
+
+#. module: l10n_it_account_stamp
+#: model:ir.model,name:l10n_it_account_stamp.model_account_invoice_line
+msgid "Invoice Line"
+msgstr "Righe Fattura"
+
+#. module: l10n_it_account_stamp
+#: field:product.template,is_stamp:0
+msgid "Is stamp"
+msgstr "È una marca da bollo"
+
+#. module: l10n_it_account_stamp
+#: model:product.template,description_sale:l10n_it_account_stamp.l10n_it_account_stamp_2_euro_product_template
+msgid "Marca da bollo da 2,00 euro. Assolvimento dell’imposta di bollo ai sensi dell’articolo 6, comma 2, del Dm 17 giugno 2014."
+msgstr ""
+
+#. module: l10n_it_account_stamp
+#: field:account.invoice.line,absolute_amount_total:0
+msgid "Amount"
+msgstr "Importo"
+
+#. module: l10n_it_account_stamp
+#: view:account.invoice:0
+msgid "CadetBlue: is_stamp_line == True;"
+msgstr ""
```

Comment:
Questa deve trovarsi nel po come traducibile? Dove compare?

Reply:
viene inserita in automatico, non è rilevante